test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,36 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
Same as above. Always remember to create a new branch **off** `master` for each enhancement.